tests/integration/testlib/ovslib.py

```diff
@@ -0,0 +1,118 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+from contextlib import contextmanager
+
+import libnmstate
+from libnmstate.schema import Interface
+from libnmstate.schema import InterfaceState
+from libnmstate.schema import InterfaceType
+from libnmstate.schema import OVSBridge
+
+
+@contextmanager
+def ovs_bridge(
+    name,
+    bridge_subtree_state=None,
+    extra_iface_state=None,
+    additional_ifaces=None,
+):
+    desired_state = {
+        Interface.KEY: [
+            {
+                Interface.NAME: name,
+                Interface.TYPE: InterfaceType.OVS_BRIDGE,
+                Interface.STATE: InterfaceState.UP,
+                OVSBridge.CONFIG_SUBTREE: bridge_subtree_state or {},
+            }
+        ]
+    }
+    if extra_iface_state:
+        desired_state[Interface.KEY][0].update(extra_iface_state)
+
+    if additional_ifaces:
+        desired_state[Interface.KEY].extend(additional_ifaces)
+
+    libnmstate.apply(desired_state)
+
+    try:
+        yield desired_state
+    finally:
+        libnmstate.apply(
+            {
+                Interface.KEY: [
+                    {
+                        Interface.NAME: name,
+                        Interface.STATE: InterfaceState.ABSENT,
+                    }
+                ]
+            },
+            verify_change=False,
```

Comment:
Like Edy, I also don't like `verify_change=False` here.  The `libnmstate` has different verification code comparing to the `assertlib` in test. It could be better we can compare both verification results.

You replied with `Done`  after https://github.com/nmstate/nmstate/pull/513#discussion_r331051989 , what does that means?

Reply:
IIUIC Edy wanted to verify it only on the configuration, not the cleanup (the same that is done in the linux bridge helper).